src/main/java/seedu/address/logic/commands/AddSchedCommand.java

```diff
@@ -65,20 +67,33 @@ public CommandResult execute(Model model) throws CommandException {
             participants.add(lastShownList.get(index.getZeroBased()));
             participantsNames.add(lastShownList.get(index.getZeroBased()).getName().toString());
         }
+        schedule.addParticipants(participantsNames);
         if (model.hasSchedule(schedule)) {
             model.addSchedulePeople(schedule, participantsNames);
-            /*for (String name : participantsNames) {
+            ObservableList<Schedule> arraySched = model.getAddressBook().getScheduleList();
+            final Schedule[] edittedSchedule = new Schedule[1];
+            arraySched.forEach(schedule1 -> {
+                if (schedule1.isSameSchedule(schedule)) {
+                    edittedSchedule[0] = schedule1;
+                }
+            });
+            System.out.println("i was called cause duplicate");
```

Comment:
Thanks for leaving these println, but do we want to keep it for the final product?